shared/actions/tracker2.tsx

```diff
@@ -303,10 +305,20 @@ const loadNonUserProfile = async (action: Tracker2Gen.LoadNonUserProfilePayload)
     return false
   } catch (e) {
     logger.warn(`Error loading non user profile: ${e.message}`)
-    return false
+    return Tracker2Gen.createErrorLoadingNonUserProfile({assertion, error: e.message})
```

Comment:
make sure to test this... last time I checked, GetNonUserDetails also bailed with error on keybase usernames, and `loadNonUserProfile` seems to be always called, even if you are not attempting to view sbs profile.

maybe my findings were wrong though

Reply:
Good catch